tests/src/python/test_qgsserver.py

```diff
@@ -466,6 +691,131 @@ def test_getLegendGraphics_layertitle(self):
+            "WIDTH": "500",
+            "CRS": "EPSG:3857"
+        }.items())])
+
+        r, h = self._result(self.server.handleRequest(qs))
+        self._img_diff_error(r, h, "WMS_GetLegendGraphic_SymbolSpace")
+
+    def test_wms_GetLegendGraphic_IconLabelSpace(self):
+        qs = "&".join(["%s=%s" % i for i in list({
+            "MAP": urllib.parse.quote(self.projectPath),
+            "SERVICE": "WMS",
+            "VERSION": "1.1.1",
+            "REQUEST": "GetLegendGraphic",
+            "LAYER": "Country,Hello",
+            "LAYERTITLE": "FALSE",
+            "ICONLABELSPACE": "100",
+            "FORMAT": "image/png",
+            "HEIGHT": "500",
+            "WIDTH": "500",
+            "CRS": "EPSG:3857"
+        }.items())])
+
+        r, h = self._result(self.server.handleRequest(qs))
+        self._img_diff_error(r, h, "WMS_GetLegendGraphic_IconLabelSpace")
+
+    def test_wms_GetLegendGraphic_SymbolSize(self):
+        qs = "&".join(["%s=%s" % i for i in list({
+            "MAP": urllib.parse.quote(self.projectPath),
+            "SERVICE": "WMS",
+            "VERSION": "1.1.1",
+            "REQUEST": "GetLegendGraphic",
+            "LAYER": "Country,Hello",
+            "LAYERTITLE": "FALSE",
+            "SYMBOLWIDTH": "50",
+            "SYMBOLHEIGHT": "30",
+            "FORMAT": "image/png",
+            "HEIGHT": "500",
+            "WIDTH": "500",
+            "CRS": "EPSG:3857"
+        }.items())])
+
+        r, h = self._result(self.server.handleRequest(qs))
+        self._img_diff_error(r, h, "WMS_GetLegendGraphic_SymbolSize")
+
+    #def test_wms_GetLegendGraphic_BBox(self):
+    #    qs = "&".join(["%s=%s" % i for i in list({
+    #        "MAP": urllib.parse.quote(self.projectPath),
+    #        "SERVICE": "WMS",
+    #        "VERSION": "1.1.1",
+    #        "REQUEST": "GetLegendGraphic",
+    #        "LAYER": "Country,Hello,db_point",
+    #        "LAYERTITLE": "FALSE",
+    #        "FORMAT": "image/png",
+    #        "HEIGHT": "500",
+    #        "WIDTH": "500",
+    #        "BBOX": "-151.7,-38.9,51.0,78.0",
+    #        "CRS": "EPSG:4326"
+    #    }.items())])
+    #    r, h = self._result(self.server.handleRequest(qs))
+    #    self._img_diff_error(r, h, "WMS_GetLegendGraphic_BBox")
+
+    #def test_wms_GetLegendGraphic_BBox2(self):
+    #    qs = "&".join(["%s=%s" % i for i in list({
+    #        "MAP": urllib.parse.quote(self.projectPath),
+    #        "SERVICE": "WMS",
+    #        "VERSION": "1.1.1",
+    #        "REQUEST": "GetLegendGraphic",
+    #        "LAYER": "Country,Hello,db_point",
+    #        "LAYERTITLE": "FALSE",
+    #        "FORMAT": "image/png",
+    #        "HEIGHT": "500",
+    #        "WIDTH": "500",
+    #        #"BBOX": "-76.08,38.04,109.95,-6.4",
+    #        "SRS": "EPSG:4326"
+    #    }.items())])
+
+    #    r, h = self._result(self.server.handleRequest(qs))
+    #    self._img_diff_error(r, h, "WMS_GetLegendGraphic_BBox2")
+
```

Comment:
Since the rebase, I didn't manage to use the getLegendGraphic request with a BBOX parameter. Each time, I have an empty response. So i disabled my tests for now.

What am i missing?